### ISSUE TITLE
summary

### ISSUE THREAD

**Issue by darcboykin1** [edoardrui0/Capstone#4]
### Criteria 1: HTML Structure  
* _Score Level:_ 4
* _Comment(s):_ Immaculate HTML structure. Your document is well formatted. The indentations clearly mark the beginnings and ending of the different sections. One thing that I would recommend you consider adding in the future is comments, marking where certain sections begin and end. It'll help the document be more readable than it already is.
 
### Criteria 2: Visual Layout for Both Desktop and Mobile Sizes
* _Score Level:_ 2
* _Comment(s):_ Your desktop size is very close to the design spec. It should be perfect if you address what I mentioned in the previous issues in regards to sizing the containers. Your mobile layout unfortunately does not meet the expectations provided in the spec. I'm referring specifically to your .cta-box and .main-box sections, which are constrained against the side of the screen when it is shrunken down to  the mobile size that your declared in your CSS: 

![Screen Shot 2019-04-30 at 11 58 17 PM](https://user-images.githubusercontent.com/23035535/57005215-025d7a80-6ba4-11e9-8e1c-8dab3a14ca9a.png)

My advice on fixing this would be to start by increasing the width of those containers to 100% when the screen is 480px wide.
 
### Criteria 3: Responsive Design (Media Queries, Responsive Units, etc.)
* _Score Level:_ 4
* _Comment(s):_ Your syntax and conciseness with the responsive design portion of your CSS is great. Good job!
 
### Criteria 4: Visual Design and Accessibility (Color Palette, Typography, Transitions, etc.)
* _Score Level:_ 2
* _Comment(s):_ I see that you discovered how to create gradients in CSS, which is not something that many beginners know how to do. That in and of itself is impressive. However, your use of it here is kind of excessive. You used gradients wherever you could at the expense of establishing any sort of color palette or scheme. Things like this, while technically impressive, can negatively impact the professional impression of an otherwise well put together site. As for your typography, I see that you defined all of your fonts in your stylesheet, which is fine, but next time be sure to define your fonts at the top of your CSS, so that fonts will be first to load, and any elements that need them will have an easier time accessing them.

**Comment by edoardrui0** [comment]
Yeah, I realised AFTER i submitted the projected that i overdid it with the gradients. i left it as is cause whatever, i was pressed against time and i wanted to throw that in. I had other plans too with the hover psuedo class and a couple animations i wants to throw in, but unfortunately my schedule didn't permit me to work as much as i wanted to on this project. Thank you for your grading and recommendations
